generators/entity-server/templates/src/main/java/package/domain/Entity.java.ejs

```diff
@@ -619,16 +619,16 @@ relationships.forEach((relationship, idx) => {
         <%_ if (relationship.otherRelationship && !relationship.reference.owned) { _%>
         if (this.<%= relationshipFieldNamePlural %> != null) {
             <%_ if (relationship.otherRelationship.reference.collection) { _%>
-            this.<%= relationshipFieldNamePlural %>.forEach(<%= otherEntityName %> -> <%= otherEntityName %>.remove<%= otherEntityRelationshipNameCapitalized %>(this));
+            this.<%= relationshipFieldNamePlural %>.forEach(i -> i.remove<%= otherEntityRelationshipNameCapitalized %>(this));
```

Comment:
I guess we can find a better name for that, right? ;)

Reply:
I was going to name it item, but I could not guarantee it won’t conflict with Item entity.

I am curious about the conflict, a plural name was conflicting with a non plural name.
But the code is broken and I don’t have time to go further, I created this workaround.

Reply:
Hum, that's interesting indeed!

Reply:
A quick alternative is to name it `<%= otherEntityName %>Item`.